.github/workflows/_reusable-release.yml

```
@@ -29,25 +29,62 @@ jobs:
+      - name: Generate SBOM
+        uses: anchore/sbom-action@v0
+        with:
+          format: spdx-json
+          output-file: sbom-source.spdx.json
+      - name: Upload SBOM
+        uses: actions/upload-artifact@v4
+        with:
+          name: sbom-source
+          path: sbom-source.spdx.json
+
   release:
```

actionlint (1.7.9)

"runs-on" section is missing in job "release"

(syntax-check)